zinc/src/main/scala/sbt/internal/inc/javac/AnalyzingJavaCompiler.scala

```diff
@@ -32,6 +32,7 @@ import xsbti.{
 
 import sbt.util.InterfaceUtil
 import sbt.util.Logger
+import xsbt.Log.debug
```

Comment:
I am looking at this again to merge into develop, and I don't know if this is the right logger to use since xsbt package is visible in the compiler bridge implementation only?

Reply:
Nice catch! Shall open a new PR that removes the import and calls `log.debug` directly.